bigcache.go

```diff
@@ -49,7 +58,12 @@ func newBigCache(config Config, clock clock) *BigCache {
 			entries: *queue.NewBytesQueue(shardSize*config.MaxEntrySize, config.Verbose),
 		}
 	}
-	return cache
+
+	return cache, nil
+}
+
+func isPowerOfTwo(number int) bool {
+	return number == int(math.Pow(2, math.Ceil(math.Log2(float64(number)))))
```

Comment:
Should be [`return ((number & (number - 1)) == 0)`](http://bits.stephan-brumme.com/isPowerOfTwo.html)